bootstrap/pkg/kfapp/ksonnet/ksonnet.go

```diff
@@ -247,89 +246,7 @@ func (ksApp *ksApp) components() (map[string]*kfdefs.KsComponent, error) {
 	return comps, nil
 }
 
-func (ksApp *ksApp) deleteGlobalResources(config *rest.Config) error {
-	apiextclientset := kftypes.GetApiExtClientset(config)
-	do := &metav1.DeleteOptions{}
-	lo := metav1.ListOptions{
-		LabelSelector: kftypes.DefaultAppLabel + "=" + ksApp.Name,
-	}
-	crdsErr := apiextclientset.CustomResourceDefinitions().DeleteCollection(do, lo)
-	if crdsErr != nil {
-		return fmt.Errorf("couldn't delete customresourcedefinitions Error: %v", crdsErr)
-	}
-	crdsByName := []string{
-		"compositecontrollers.metacontroller.k8s.io",
-		"controllerrevisions.metacontroller.k8s.io",
-		"decoratorcontrollers.metacontroller.k8s.io",
-		"applications.app.k8s.io",
-	}
-	for _, crd := range crdsByName {
-		do := &metav1.DeleteOptions{}
-		dErr := apiextclientset.CustomResourceDefinitions().Delete(crd, do)
-		if dErr != nil {
-			log.Errorf("could not delete %v Error %v", crd, dErr)
-		}
-	}
-	clientset := kftypes.GetClientset(config)
-	crbsErr := clientset.RbacV1().ClusterRoleBindings().DeleteCollection(do, lo)
-	if crbsErr != nil {
-		return fmt.Errorf("couldn't get list of clusterrolebindings Error: %v", crbsErr)
-	}
-	crbName := "meta-controller-cluster-role-binding"
-	dErr := clientset.RbacV1().ClusterRoleBindings().Delete(crbName, do)
-	if dErr != nil {
-		log.Errorf("could not delete %v Error %v", crbName, dErr)
-	}
-	crsErr := clientset.RbacV1().ClusterRoles().DeleteCollection(do, lo)
-	if crsErr != nil {
-		return fmt.Errorf("couldn't delete clusterroles Error: %v", crsErr)
-	}
-	return nil
-}
-
 func (ksApp *ksApp) Delete(resources kftypes.ResourceEnum) error {
-	config := kftypes.GetConfig()
```

Comment:
why is Delete doing nothing?

Reply:
because we do everything in Gcp :p

Reply:
but isn't there a use case that I already have a k8s cluster, and I only init/generate k8s part, not platform part? So delete should only delete the k8s part?

Reply:
I haven't had such an user case before, could you elaborate more?  In general, as long as we have enough information (project/location/deployment), `kfctl delete` will try to delete everything it could find on GCP.  and currently we only assume GCP use case.  Therefore, as long as we could find project/location/deployment from app.yaml, kfctl could work.

Reply:
```
kfctl generate --help
Generate a kubeflow application where resources is one of 'platform|k8s|all'.

  platform: non kubernetes resources (eg --platform gcp)
  k8s: kubernetes resources
  all: both platform and k8s
```

It should be able to do k8s only

Reply:
we only assume GCP on 0.5 kfctl-go